benchmark/runner/conf/benchmark/savina_concurrency_barber.yaml

```diff
@@ -5,6 +5,7 @@ params:
   waiting_room_size: 1000
   production_rate: 1000
   haircut_rate: 1000
+  num_iterations: 12
```

Comment:
The list here is intended for benchmark specific parameters. `num_iterations` and `threads` are global parameters that apply to all benchmark. Therefore, `num_iterations` should not be listed here and also not be passed to the `gen_args`. This is taken care of automatically.

Note that this comment applies to all the changed yaml files.